README.adoc

```diff
@@ -198,20 +199,26 @@ include::{common-includes}/mvnpackage.adoc[]
 
 == Existing Tracer implementation
 
-In order to collect traces across your systems, you are required to implement the OpenTracing `Tracer`
+To collect traces across your systems, you are required to implement the OpenTracing `Tracer`
 interface. For the purposes of this guide, we are providing a bare-bones `Tracer` implementation for
 the Zipkin server in a form of a user feature for Open Liberty.
 
-This feature has already been configured for you and it will be installed automatically into each service
-when you build them. You can find it enabled in your `server.xml` files:
+This feature is already configured for you in your `pom.xml` and `server.xml` files. It will be
+downloaded and installed automatically into each service when you run a Maven build. You can find it
+enabled in your `server.xml` files as:
+
+// This feature is already configured for you and it will be installed automatically into each service
+// when you build them. You can find it enabled in your `server.xml` files:
```

Comment:
How about, "...when you build each one. You can find the feature enabled..."?

Reply:
same as above